script/build.ts

```diff
@@ -149,24 +176,30 @@ function copyStaticResources() {
 }
 
 function copyDependencies() {
-  const originalPackage = require(path.join(projectRoot, 'app', 'package.json'))
+  const originalPackage: Package = require(path.join(
+    projectRoot,
+    'app',
+    'package.json'
+  ))
 
   const commonConfig = require(path.resolve(__dirname, '../app/webpack.common'))
   const externals = commonConfig.externals
   const oldDependencies = originalPackage.dependencies
-  const newDependencies = {}
+  const newDependencies: PackageLookup = {}
 
-  for (var [name, spec] of Object.entries(oldDependencies)) {
+  for (const name of Object.keys(oldDependencies)) {
```

Comment:
I couldn't get TS to light up `Object.entries` despite it being part of `es2017.object` as per https://github.com/Microsoft/TypeScript/pull/8538 _\*shrug\*_